### ISSUE TITLE
Could be this used for reverse proxy ? 

### ISSUE THREAD

**Issue by vdhub** [jlesage/docker-nginx-proxy-manager#274]
The idea is similar to caddy i guess where i have subdomain.domain.office and each time someone in the office access the domain.office and it's subdomains to show the proper green certificate instead of the red warning popup on any browser. 

IF it can be done, then how and what do i need to configure. 

This can be added on a Synology DS1817+ over Docker ?

Currently it is running DSM 7.1.1-42962 Update 3 ; Docker 20.10.3-1308   

Thanks in advance.

**Comment by Hukuma1** [comment]
Yes. This is a reverse proxy with a nice GUI. Set it up in Docker and enjoy.

**Comment by vdhub** [comment]
well i forgot to mention that my public ip is dynamic and i don’t want that insecure certificate popup to show on browsers, so how and what can i do in this case ? any idea where to look ? 

thanks in advance

**Comment by Hukuma1** [comment]
https://www.youtube.com/watch?v=bQdqf5xAyUk

You need a domain name and a way to update your IP to it. 

Or a service like DuckDNS works as well: https://www.youtube.com/watch?v=wrMn8sar-nA

**Comment by vdhub** [comment]
I'm not sure i was clear enough 

I have the fdqn : domain.com 
I have the ddns account that links my dynamic ip to a subdomain.domain.com 
i have several domains, which are local : like it.domain.local; app.domain.local

Now, i don't need to access it.domain.local; app.domain.local from internet , i need those to be accessed only from within the LAN but i don't want to see that certificate warning. So the idea was to use something to act like a CA or intermediate CA. that will sign the it.domain.local; app.domain.local certificates and showing them as valid and green and not showing that red warning on browsers . 

my 2 subdomains should be accessible on LAN only . 

Thanks.

**Comment by vdhub** [comment]
i think i may have found the solution, if i can get it work as well. https://caddyserver.com/docs/automatic-https#wildcards
